cmd/nri-ecs_test.go

```diff
@@ -43,21 +40,62 @@ func TestIntegrationPublish(t *testing.T) {
 		os.Stdout = stdout
 	}()
 
-	main()
+	cases := map[string]struct {
+		response       string
+		endpointEnvVar string
+	}{
+		"from_v3_metadata_endpoint": {
+			response:       v3,
+			endpointEnvVar: metadata.ContainerMetadataEnvVar,
+		},
+		"from_v4_metadata_endpoint": {
+			response:       v4,
+			endpointEnvVar: metadata.ContainerMetadataV4EnvVar,
+		},
+	}
```

Comment:
would make sense to add a no-happy path returning garbage data or no data? Is there already a test covering it?

Reply:
I don't think so , i will add one.